src/components/playlistModal/RemovePlaylistModal.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Collection } from 'boclips-api-client/dist/sub-clients/collections/model/Collection';
+import { Bodal } from 'src/components/common/bodal/Bodal';
+import { Typography } from '@boclips-ui/typography';
+import { useNavigate } from 'react-router-dom';
+import { useRemovePlaylistMutation } from 'src/hooks/api/playlistsQuery';
+import s from './style.module.less';
+
+export interface Props {
+  playlist: Collection;
+  onCancel: () => void;
+  showErrorNotification: (message: string, dataQa: string) => void;
+}
+
+export const RemovePlaylistModal = ({
+  playlist,
+  onCancel,
+  showErrorNotification,
+}: Props) => {
+  const navigate = useNavigate();
+
+  const {
+    mutate: removePlaylist,
+    isSuccess,
+    isError,
+  } = useRemovePlaylistMutation(playlist);
+
+  React.useEffect(() => {
```

Comment:
👋 I'm just being nosey so feel free to ignore my comments, but you could remove the need for a useEffect here and use onSuccess and onError callbacks from the useMutation hook - https://react-query-v3.tanstack.com/guides/mutations#mutation-side-effects 

This https://tkdodo.eu/blog/mastering-mutations-in-react-query#some-callbacks-might-not-fire suggests doing something like  
```
// null looks a bit weird here though </3
removePlaylist(null, {
    onSuccess: () => navigate('/playlists'), 
    onError: () => showErrorNotification(
        `Error: Failed to remove playlist "${playlist.title}"`,
        'remove-playlist-failed',
      )
})
```